src/charm.py

```diff
@@ -940,8 +940,14 @@ def on_fio_action(self, event):
             )
             while (datetime.datetime.now() < test_end):
                 _result = json.loads(_bench.fio(_fio_conf))
+                if event.params["operation"] in ["read", "randread"]:
+                    metrics = ["read"]
+                elif event.params["operation"] in ["write", "randwrite"]:
+                    metrics = ["write"]
+                else:
+                    metrics = ["read", "write"]
                 for job in _result["jobs"]:
-                    for metric in ('read', 'write'):
+                    for metric in metrics:
```

Comment:
Yes it is.
FIO will be able to output the results on both Jammy and Focal.
It would have always worked on Focal, this modification was required specifically on Jammy because some fields have disappeared.